### PR TITLE
Add 2019-03-26 planning call and update schedule

### LINKED ISSUE #27: Update organizer meeting schedule
Now that we're settling on a later date #9, we should relax our meeting schedule in Apr/May/Jun and think about what we need for timing over the summer

### PR DESCRIPTION
**Pull request by dcwalk**
Notes from planning call and closes #27

### CODE REVIEW

File: notes/2019-03-16-planning-call.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    - Kickoff Fri Sep 20 with alcohol (format likely similar to last year)
+- Dawn to follow up this week
+
+## Website Check-in
+
+- Support us
+    - amts fine: $15 stickers! $30 tote! 
+    - Dawn generate paypal thing
+    - updates to page: layout of tiers, tote image why?!
+- Revisting dweb publish flow
```
Comment (garrying) [inline notes/2019-03-16-planning-call.md]:
```suggestion
- Revisiting dweb publish flow
```

File: notes/2019-03-16-planning-call.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+- No deadline, rolling acceptance
+- Is flow async work and then meeting to finalize?
+
+## Sponsorship
+
+- Draft package:
+https://docs.google.com/document/d/1IGuBnd3O1CnICe8baFD8WSF9cWZjjls5-6zE8qCEDEM/edit#heading=h.ofdcikls6w3z
+- Nice to start reaching out with date and venue posted
+- CFP before official push? (align with social media push)
+- Some friendly potential sponsers in Mozilla before CFP?
```
Comment (garrying) [inline notes/2019-03-16-planning-call.md]:
```suggestion
- Some friendly potential sponsors in Mozilla before CFP?
```

File: notes/2019-03-16-planning-call.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    - 2/mth throughout Apr-Jun
+    - Jul busy for at least a couple folks
+    - End JUL before break important to connect
+    - Last week August (August 20 on) meet regularly until load-in
+
+## Venue Update
+
+- Agreement back with Jennie for update, the modification to notes section:
+    - Exhibition/conference installation Sep 19, 10am-6pm
+    - Exhibition open daily, Sep 20-24, 10am-6m
```
Comment (garrying) [inline notes/2019-03-16-planning-call.md]:
```suggestion
    - Exhibition open daily, Sep 20-24, 10am-6pm
```

**Comment by dcwalk** [comment]
Tanks @garrying :joy: my typos :grimacing:

**Pull request merged**